src/scanner/scanCurrent.js

```diff
@@ -0,0 +1,299 @@
+const { Api } = require('@cennznet/api');
+require("dotenv").config();
+const logger = require('../logger');
+const { NftWallet, NftListing } = require('../mongo/models');
+const mongoose = require('mongoose');
+
+async function processExtrinsicData(extrinsics) {
+    await Promise.all(
+        extrinsics.map(async (extrinsic) =>{
+            const args = extrinsic.method.args;
+            switch (extrinsic.method.method) {
+                case 'burnBatch': {
+                    const owner = extrinsic.signer;
+                    const collectionId = args[0];
+                    const seriesId = args[1];
+                    const serialNumbers = args[2];
+                    await burnWalletNFT(owner, collectionId, seriesId, serialNumbers);
+                }
+                case 'burn':{
+                    console.log('args:',args);
+                    const token = args[0];
+                    const owner = extrinsic.signer;
+                    console.log('token:', token);
+                    console.log('owner:', owner);
+                    await burnWalletNFT(owner, token[0], token[1], [token[2]]);
+                }
+            }
+        })
+    );
+}
+
+async function processEventData(dataFetched, method, api) {
+    logger.info('Event triggered::',method);
+    switch (method) {
+        case 'CreateToken': {
+            const tokenId = dataFetched[1];
+            const owner = dataFetched[2];
+            await saveWalletData(owner, [], [tokenId]);
+            break;
+        }
+        case 'CreateSeries': {
+            const collectionId = dataFetched[0];
+            const seriesId = dataFetched[1];
+            const quantity = dataFetched[2];
+            const owner = dataFetched[3];
+            let tokens = [];
+            for (let i = 0; i < quantity; i++) {
+                tokens.push([collectionId, seriesId, i]);
+            }
+            await saveWalletData(owner, [], tokens);
+            break;
+        }
+        case 'CreateAdditional': {
+            const collectionId = dataFetched[0];
+            const seriesId = dataFetched[1];
+            const quantity = dataFetched[2];
+            const owner = dataFetched[3];
+            let tokens = [];
+            const currentNextIndex = await api.query.nft.nextSerialNumber(collectionId, seriesId);
```

Comment:
nit: currnetNextIndex confusing name
```suggestion
            const nextIndex = await api.query.nft.nextSerialNumber(collectionId, seriesId);
```

Reply:
done